runtime/parachains/src/inclusion.rs

```diff
@@ -1394,7 +1423,8 @@ pub(crate) mod tests {
 						vec![signed.into()],
 						DisputedBitfield::zeros(expected_bits()),
 						&core_lookup,
-					),
+					)
+					.0,
```

Comment:
honestly, I would make these function self-weighing to prevent such breakage. Any advantage in returning the weight?

Reply:
@kianenigma the main advantage I see is explicitness when reading at the top level. Otherwise it may not be clear to the reader of `enter` that this weighing is going on. 

But I think self weighing sounds like it might be a good approach. How would I go about doing that?

Reply:
> How would I go about doing that?

Instead of returning anything, each function calls into `frame_system::Pallet::register_extra_weight` itself. 

I don't strongly prefer this, chose whichever approach suits your need best. Self-weighing is def. less noise, but that noise could be for a good reason.